frontends/election-manager/src/screens/machine_locked_screen.tsx

```diff
@@ -11,7 +11,7 @@ import { AppContext } from '../contexts/app_context';
 
 const LockedImage = styled.img`
   margin-right: auto;
-  margin-bottom: 5px;
+  margin-bottom: 1.25em;
```

Comment:
I've always felt that the lock icon was too close to the text below it, so I gave it some breathing room. I make the same change in VxCentralScan later in this PR.

_Before on the left, after on the right_

<img src="https://user-images.githubusercontent.com/12616928/183748427-fc74f0e3-e731-4bd2-aed9-af4383e872ec.png" alt="before" width=300 /> <img src="https://user-images.githubusercontent.com/12616928/183748426-9b421111-fcf4-42b3-8d1a-9b0787ed2d07.png" alt="after" width=300 />

Let me know if you have any concerns with this @beausmith!